src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -69,19 +53,11 @@ public PaxTarEntry(TarEntryType entryType, string entryName)
         /// <item>In all platforms: <see cref="TarEntryType.Directory"/>, <see cref="TarEntryType.HardLink"/>, <see cref="TarEntryType.SymbolicLink"/>, <see cref="TarEntryType.RegularFile"/>.</item>
         /// <item>In Unix platforms only: <see cref="TarEntryType.BlockDevice"/>, <see cref="TarEntryType.CharacterDevice"/> and <see cref="TarEntryType.Fifo"/>.</item>
         /// </list>
-        /// The specified <paramref name="extendedAttributes"/> get appended to the default attributes, unless the specified enumeration overrides any of them.
-        /// <para>The following entries are always found in the Extended Attributes dictionary of any PAX entry:</para>
+        /// The specified <paramref name="extendedAttributes"/> are additional attributes to be used for the entry.
+        /// <para>It may include PAX attributes like:</para>
```

Comment:
I've updated this list to the things I think a user may want to set.
For the other items mentioned, `TarEntry` has properties which control what is effectively written.